avd_docs/aws/apigateway/AVD-AWS-0179/Terraform.md

```diff
@@ -0,0 +1,29 @@
+
+Enable cache
+
+```hcl
+ resource "aws_api_gateway_rest_api" "example" {
+	
+ }
+
+ resource "aws_api_gateway_stage" "example" {
+
+ }
+
+ resource "aws_api_gateway_method_settings" "good_example" {
+   rest_api_id = aws_api_gateway_rest_api.example.id
+   stage_name  = aws_api_gateway_stage.example.stage_name
+   method_path = "path1/GET"
+ 
+   settings {
+     metrics_enabled = true
+     logging_level   = "INFO"
+     caching_enabled = true
+   }
+ }
+ 
+```
+
+#### Remediation Links
+ - https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/api_gateway_method_settings#cache_enabled
+
```

Comment:
ditto

Reply:
I deleted these files, But these files are auto generated by running make docs. Am i doing something wrong?

Reply:
Not sure, there's a few rules which seem to have this problem (I'm opening a PR now to fix that) but not these ones. Make sure your feature branches are up to date with the master.